lib/teams_manager.rb

```diff
@@ -16,36 +16,23 @@ def generate_list(data_path)
     list_of_data
   end
 
-  def count_of_teams #ask intructors if they want this
+  def count_of_teams
     teams.uniq.count
   end
 
-  def get_names_hash
+  def get_team_name(team_id)
     team_names_hash = {}
     @teams.each do |team|
       team_names_hash[team.team_id] = team.team_name
     end
```

Comment:
Yes, makes a lot of sense to simply combine, and save one mostly unused method.

Reply:
This one is excellent.